_download-meta/openttd-nightlies.md

```diff
@@ -1,15 +1,14 @@
 ---
-folder_old_infrastructure: nightlies/trunk
 ---
 
 OpenTTD requires a separate base graphics set to run, which can be downloaded through the installer (Windows) or on first run (Windows, Linux).
 In case you want to manually download and install it, some options are:
-* [OpenGFX (graphics)](../opengfx-releases/latest.html)
+* [OpenGFX (graphics)](https://www.openttd.org/downloads/opengfx-releases/latest.html)
```

Comment:
These explicit URLs will break the staging site, won't they? Can `{{ site.url }}` be used instead?

Reply:
or `page.host`, which was used in a few places previously

Reply:
Two problems here:
1) these files cannot use variables in the regular sense. This is why those @@ are there. At least, I haven't found a way :D
2) when building the site, it is unknown if it will be running on staging or production. So there is no way to get these URLs correct.

Now it was the reason they used to be relative, but that was giving other nasty problems when I was using this to generate HTML files on other URLs. But as I no longer do that, I can make them absolute URLs :D Fixing ...